javascript/builtins/Error.json

```diff
@@ -105,6 +105,58 @@
             }
           }
         },
+        "cause": {
+          "__compat": {
+            "description": "<code>Error.cause</code> represents additional contextual information about an error for the purpose of chaining errors.",
```

Comment:
We can omit this description, so it'll show up like other properties of `Error`.

I'd also expect a corresponding subfeature of `Error()`, showing whether `options.cause` is supported.

```suggestion
```